SemanticUI-less blocks widget (ObjectListWidget) + Improve and reorder documentation using diataxis. (#486)
kitconcept/volto-light-theme: docs/reference/colors.md

# Colors

> 

Reviewer: ```suggestion

% TODO empty heading needs content

```

## Theme colors

> 

Reviewer: ```suggestion

% TODO empty heading needs content

```

## Color definition syntax

The VLT has migrated to use the standardized color definitions in Volto.
These definitions use CSS properties that are injected at runtime in the right places, so your CSS can adapt to use them in a generic way.
The resultant CSS is simpler, and there's no need to define class names for each color definition.

> Read more about them in: https://6.docs.plone.org/volto/development/color-picker-widget.html?highlight=color#custom-css-properties-as-color-definitions

Reviewer: ```suggestion
Read more about them in the official Plone documentation, [Custom CSS properties as color definitions](https://6.docs.plone.org/volto/development/color-picker-widget.html#custom-css-properties-as-color-definitions).
```

> The color definitions are stored in `config.blocks.themes` and you can extend them from your project or add-ons.

Reviewer: ```suggestion
The color definitions are stored in `config.blocks.themes`, and you can extend them from your project or add-ons.
```
